### PR TITLE
Remove UNDERFS_ADDRESS PropertyKey

### LINKED ISSUE #8384: Remove UNDERFS_ADDRESS PropertyKey
**Alluxio Version:** 1.8.1

**Describe the bug**
The `UNDERFS_ADDRESS` PropertyKey has been deprecated since 1.5.0 and should be removed for version 2.0 

**To Reproduce**
N/A

**Expected behavior**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by ZacBlanco**
Removed all occurrences of UNDERFS_ADDRESS in the
code and documentation. Any places which needed
replacements now use MASTER_MOUNT_TABLE_ROOT_UFS.
In the case of environment variables we make sure that
"ALLUXIO_" is prepended.

Fixes #8384

### CODE REVIEW

File: docs/cn/basic/Configuration-Settings.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Alluxio通过环境变量来支持一些常用的配置设置，包括:
   <td>从1.1版本开始被<code class="highlighter-rouge">ALLUXIO_MASTER_HOSTNAME</code>替代，并将在2.0版本中移除</td>
 </tr>
 <tr>
-  <td><code class="highlighter-rouge">ALLUXIO_UNDERFS_ADDRESS</code></td>
+  <td><code class="highlighter-rouge">MASTER_MOUNT_TABLE_ROOT_UFS</code></td>
```
Comment (madanadit) [inline docs/cn/basic/Configuration-Settings.md]: missed prefix ALLUXIO_

File: integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java [hunk 2]
```diff
@@ -167,8 +167,9 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
                             .newBuilder()
                             .addVariables(
                                 Protos.Environment.Variable.newBuilder()
-                                    .setName("ALLUXIO_UNDERFS_ADDRESS")
-                                    .setValue(ServerConfiguration.get(PropertyKey.UNDERFS_ADDRESS))
+                                    .setName("MASTER_MOUNT_TABLE_ROOT_UFS")
```
Comment (madanadit) [inline integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java]: missed prefix

File: integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java [hunk 3]
```diff
@@ -212,8 +213,9 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
                                     .build())
                             .addVariables(
                                 Protos.Environment.Variable.newBuilder()
-                                    .setName("ALLUXIO_UNDERFS_ADDRESS")
-                                    .setValue(ServerConfiguration.get(PropertyKey.UNDERFS_ADDRESS))
+                                    .setName("MASTER_MOUNT_TABLE_ROOT_UFS")
```
Comment (madanadit) [inline integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java]: missed prefix

Comment (ZacBlanco) [reply integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java]: fixed

File: integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java (hunk 2 above)
Comment (ZacBlanco) [reply integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java]: fixed

File: docs/cn/basic/Configuration-Settings.md (hunk 1 above)
Comment (ZacBlanco) [reply docs/cn/basic/Configuration-Settings.md]: fixed

**Pull request merged**